src/app/globals.scss

```diff
@@ -50,12 +50,6 @@ html, body {
 	width: 100%;
```

Comment:
You have to many redundant classes in this file. 

For reset styles you would normally use reset.scss and you would import reset.scss in global.scss

https://stackoverflow.com/questions/11578819/css-reset-what-exactly-does-it-do